Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM clever/kinesis-cwlogs-splitter:9cdf4ca
```

Comment:
For the moment, I'd like to continue iterating on this simpler setup just to verify it's possible, but I can add in other files before merge.